xbmc/XBDateTime.cpp

```diff
@@ -853,7 +853,7 @@ std::string CDateTime::GetAsDBTime() const
   SYSTEMTIME st;
   GetAsSystemTime(st);
 
-  return StringUtils::Format("%02i:%02i:%02i", st.wHour, st.wMinute, st.wSecond);
+  return StringUtils::Format("%02i:%02i", st.wHour, st.wMinute);
```

Comment:
Why is this change needed?

Reply:
this is only used by:
https://github.com/xbmc/xbmc/blob/eb161cdb7c4c97e9f054192c9ea9e8d00839c305/xbmc/settings/SettingDateTime.h#L43

The setValue call expects a string in the format `HH:MM` for settings with type `time`. If you prefer I can leave it as  before and only send a substr when calling setValue.

Reply:
IMO you should add a separate method or an argument to choose whether to return with or without seconds (defaulting to with seconds) for this. Because now there is an inconsistency between `CDateTime::GetAsDbTime()` and `CDatetime::GetAsDBDateTime()` which still returns the time in `HH:MM:SS` format.

Reply:
Thanks @Montellese, nice to see you back.
Yeap I though about that inconsistency. Will do.